src/mount/mount-tool.c

```diff
@@ -680,28 +696,194 @@ static int stop_mount(
         if (!arg_quiet)
                 log_info("Stopped unit %s%s%s for mount point: %s%s%s",
                          ansi_highlight(), mount_unit, ansi_normal(),
-                         ansi_highlight(), arg_mount_where, ansi_normal());
+                         ansi_highlight(), where, ansi_normal());
 
         return 0;
 }
 
 static int stop_mounts(
                 sd_bus *bus,
-                char **argv) {
+                char *where,
+                bool ignore_enoent) {
 
         int r;
 
-        r = stop_mount(bus, argv + optind, ".mount");
+        path_kill_slashes(where);
```

Comment:
sorry I'd didn't realize this earlier, but I really dislike functions that change their input parameters, except where that is obviously the right thing to do. hence I think any path_kill_slashes() invocation should be either done by the caller here, or we should operate on a copy (maybe using strdupa()) here